nutribest.com/Admin-Side/Products-Cat/products.jsx

```diff
@@ -1,361 +0,0 @@
-import {
-  Avatar,
-  Box,
-  Button,
-  Container,
-  Heading,
-  Image,
-  Input,
-  Modal,
-  ModalBody,
-  ModalCloseButton,
-  ModalContent,
-  ModalFooter,
-  ModalHeader,
-  ModalOverlay,
-  Select,
-  SimpleGrid,
-  Text,
-  useDisclosure,
-  useToast,
-} from "@chakra-ui/react";
-import React, { useEffect, useState } from "react";
-import ProductItems from "./products-item";
-import axios from "axios";
-
-const Products = () => {
-  const toast = useToast();
-  const [datas, setDatas] = useState([]);
```

Comment:
What is this 'datas' state determining? Always give meaningful names to your states. For example if this is product data, then this state name should be [productData, setProductData]